app/src/main.tsx

```diff
@@ -4,19 +4,17 @@ import App from "./App";
 
 console.debug(import.meta);
 
+const mock = true;
```

Comment:
Whether to enable mocking. Leave as true for now. This can eventually be toggled by something else, e.g. a URL param like we did in monarch. If doing `if (true)`, eslint/ts complains about useless conditional, so extract as var.

Reply:
Nice; I actually like having this extracted as a variable so it can be quickly toggled, versus looking for an `if (true)` statement somewhere in the code. I also like the idea of having it being toggled without having to change the code; perhaps it could start as an env var and also get its value from the URL later, as you suggested.